setupext/test_java.py

```diff
@@ -27,6 +28,8 @@ def run(self):
         distutils.log.info("Building Java testbench")
         buildXmlFile = os.path.join("test", "build.xml")
         command = [self.distribution.ant, '-f', buildXmlFile]
+        if self.distribution.ant.endswith(".py"):
+            command.insert(0, sys.executable)
```

Comment:
wouldn't it be easier to set ant.py executable with a shebang? Dunno if this works on windows though (today I'm gonna remove every trace of this OS from my personal computer *happy*, farewell win7)

Reply:
https://docs.python.org/3/using/windows.html#shebang-lines

Reply:
Well given every time I try to do anything on the CI it burns 2-4 hours due to having to replicate the problem without terminal, lets just remove ant from the equation entirely.   We still need to keep the ant files as they are used by LGTM and some other tools, but there is no reason to require them to build.